service/src/main/java/bio/terra/workspace/app/controller/ControlledAwsResourceApiController.java

```diff
@@ -77,6 +81,8 @@ public class ControlledAwsResourceApiController extends ControlledResourceContro
 
   private final Logger logger = LoggerFactory.getLogger(ControlledAwsResourceApiController.class);
 
+  private final WsmResourceService wsmResourceService;
```

Comment:
consider moving WsmResourceService to ControlledResourceControllerBase

Reply:
I want to go the opposite direction. I think everything in ControlledResourceControllerBase should be in a "utils" module. There isn't a need for a base class.